app/src/migrations/20220407224032-add-acronym-ministry.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
```

Comment:
Don't think we explicitly need this clause in our node execution environments, but doesn't hurt either I suppose.